docs/sdkx-unreal/sdk-configuration.mdx

```diff
@@ -80,6 +82,64 @@ The default value of this flag is `true` i.e., the in-app notification will be e
 
 </div>
 
+## FAQ Filtering by tags {#faq-filtering}
```

Comment:
Is this really applicable to SDK X? I think it's a legacy SDK only feature.

Reply:
Yes, it is applicable to SDK X.
It is already present in SDK X android, iOS , cocos sections